### PR TITLE
Pathways base

### PR DESCRIPTION
**Pull request by mattb-hippo**

### CODE REVIEW

File: Childrens-Social-Care-CPD/Views/Shared/_Content.cshtml [hunk 1]
```diff
@@ -29,4 +29,25 @@
     }
 
     model.ContentStack.Pop();
+
+    if (Model.PageType == PageType.PathwaysContentsPage)
```
Comment (mattb-hippo) [inline Childrens-Social-Care-CPD/Views/Shared/_Content.cshtml:33]: This series of if statements would be better as a switch

File: Childrens-Social-Care-CPD/Views/Shared/_Header.cshtml [hunk 2]
```diff
@@ -69,6 +69,10 @@
                     {
                         RenderMenuItem("agencyResources", "Agency resources", "agency-resources", category == "Agency resources");
                     }
+                    if (featuresConfig.IsEnabled(Features.PathwaysTraining))
+                    {
+                        RenderMenuItem("pathwaysTraining", "Pathways training", "pathways-social-work-leadership-modules-available", category == "Pathways training");
```
Comment (mattb-hippo) [inline Childrens-Social-Care-CPD/Views/Shared/_Header.cshtml:74]: Query: Is this the right URL for the top-level navigation?  Should it be the 'modules available' page or the 'all pathways overview' page.

File: Childrens-Social-Care-CPD/Views/Shared/_ErrorLayout.cshtml [hunk 3]
```diff
@@ -107,7 +107,18 @@
...
+                                <svg class="dfe-icon dfe-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" width="34" height="34">
+                                    <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+                                </svg>
+                            </a>
+                        </li>
+                    }
+                    @if (featuresConfig.IsEnabled(Features.PathwaysTraining))
+                    {
+                        <li class="dfe-header__navigation-item" id="mmi-pathwaysTraning">
+                            <a class="dfe-header__navigation-link" href="/pathways-social-work-leadership-modules-available">
```
Comment (mattb-hippo) [inline Childrens-Social-Care-CPD/Views/Shared/_ErrorLayout.cshtml:120]: Query: Is this the right URL for the top-level navigation? Should it be the 'modules available' page or the 'all pathways overview' page.

File: Childrens-Social-Care-CPD/Views/Shared/_PageBanner.cshtml [hunk 4]
```diff
@@ -4,17 +4,31 @@
...
             <div class="govuk-grid-row govuk-!-padding-top-3 govuk-!-padding-bottom-0">
                 <div class="govuk-grid-column-two-thirds-from-desktop banner-breadcrumbs">
-                    <partial name="_BreadcrumbTrail" />
+                    <partial name="_BreadcrumbTrail" model="Model" />
                 </div>
                 <div class="govuk-grid-column-two-thirds-from-desktop govuk-!-padding-top-3">
                     <h1 id="content-banner-title" class="govuk-heading-xl">@Model.ContentTitle</h1>
                     <p id="content-banner-subtitle" class="govuk-body-l">@Model.ContentSubtitle</p>
                 </div>
+                @if (Model.PageType == PageType.PathwaysTrainingContent)
```
Comment (mattb-hippo) [inline Childrens-Social-Care-CPD/Views/Shared/_PageBanner.cshtml:19]: Query - should this location section be separated out into a partial of it's own? Or are we happy with the code living in the Page Banner partial?  The breadcrumb trail is in its own pathway ...

File: Childrens-Social-Care-CPD/Views/Shared/_Content.cshtml (hunk 1 above)
Comment (mattb-hippo) [reply Childrens-Social-Care-CPD/Views/Shared/_Content.cshtml]: Refactored now

File: Childrens-Social-Care-CPD/Views/Shared/_ErrorLayout.cshtml (hunk 3 above)
Comment (Maria-C1) [reply Childrens-Social-Care-CPD/Views/Shared/_ErrorLayout.cshtml:120]: Checked with Dave - this link is incorrect and should take the user to the main Overview page. I'll correct it.

File: Childrens-Social-Care-CPD/Views/Shared/_Header.cshtml (hunk 2 above)
Comment (Maria-C1) [reply Childrens-Social-Care-CPD/Views/Shared/_Header.cshtml]: Amended

File: Childrens-Social-Care-CPD/Views/Shared/_PageBanner.cshtml (hunk 4 above)
Comment (mattb-hippo) [reply Childrens-Social-Care-CPD/Views/Shared/_PageBanner.cshtml:19]: Have separated this out into separate partial

**Review by Maria-C1 — APPROVED** [review]

**Pull request merged**